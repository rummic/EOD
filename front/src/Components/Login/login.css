

.loginbox{
    width: 40%;
    background-color:rgba(234, 234, 234, 1);
    justify-content: center;
    margin: auto;
    margin-top: 10%;
    border-radius: 10px;
}

.login-content-box{
    width: 70%;
    padding: 30px;
    margin: auto;

}
.login-content-box button{
    width: 50%;
    margin: auto;
    margin-top: 30px;
    font-size: 20px;
}

.login-content-box input{
    font-size: 20px;
}
.login-content-box label{
    font-size: 20px;
    margin-bottom: -10px;
}

