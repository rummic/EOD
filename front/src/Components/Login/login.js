import React from 'react';


function App() {
  return (
    <div className="App">
      logowanie
    </div>
  );
}

export default App;