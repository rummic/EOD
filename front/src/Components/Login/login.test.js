import React from 'react';
import { shallow } from 'enzyme';
import login from './login.js';

describe('login', () => {
    test('Rendering login component', () => {
        const wrapper = shallow(<login />);
        expect(wrapper).toMatchSnapshot();
    });
});  

/*describe('Test case for testing login',() =>{
let wrapper;
test('login check with right data',()=>{
wrapper = shallow(<login/>);
wrapper.find('input[type="text"]').simulate('change', {target: {name: 'login', value: 'superadmin'}});
wrapper.find('input[type="password"]').simulate('change', {target: {name: 'password', value: 'admin'}});
wrapper.find('button').simulate('click');
expect(wrapper.state('isLogined')).toBe(true);
})
})*/