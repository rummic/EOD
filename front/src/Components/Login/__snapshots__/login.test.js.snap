// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`login Rendering login component 1`] = `ShallowWrapper {}`;
